### PR TITLE
Translate intro in the basics section

### PR DESCRIPTION
**Pull request by thatdaniboy**
Done issue #41

### CODE REVIEW

File: LanguageGuide/TheBasics.md [hunk 1]
```diff
@@ -3,17 +3,19 @@
...
 
-Like C, Swift uses variables to store and refer to values by an identifying name. Swift also makes extensive use of variables whose values can’t be changed. These are known as constants, and are much more powerful than constants in C. Constants are used throughout Swift to make code safer and clearer in intent when you work with values that don’t need to change.
+Como C, Swift usa variáveis para guardar e referenciar valores com um nome identificador. Swift também faz uso extensivo de variáveis cujos valores não podem ser mudados. Estas são conhecidas como constantes, e são muito mais poderosas do que as constantes em C. Constantes são usadas em Swift para fazer o código mais seguro e de objetivo mais claro quando você trabalha com valores que não precisam mudar.
 
 In addition to familiar types, Swift introduces advanced types not found in Objective-C, such as tuples. Tuples enable you to create and pass around groupings of values. You can use a tuple to return multiple values from a function as a single compound value.
```
Comment (trastoan) [inline LanguageGuide/TheBasics.md]: Faltou deletar isso daqui

Comment (trastoan) [inline LanguageGuide/TheBasics.md]: Faltou deletar isso

**Review by trastoan — CHANGES_REQUESTED** [review]

**Comment by jessicalewinter** [comment]
@thatdaniboy are you still going to translate this topic?

**Review by mateusrodriguesxyz — APPROVED** [review]

**Review by trastoan — APPROVED** [review]
Good to go

**Pull request merged**